ArcGISRuntimeSDKQt_QMLSamples/Analysis/AnalyzeHotspots/AnalyzeHotspots.qml

```diff
@@ -195,28 +179,13 @@ Rectangle {
         TextField {
             id: toDate
             width: parent.width
-            text: toThisDate.toLocaleString(Qt.locale(), "d MMM yyyy")
-            selectByMouse: true
-
-            Image {
-                anchors {
-                    verticalCenter: parent.verticalCenter
-                    right: parent.right
-                    margins: 5
-                }
-                source: "qrc:/Samples/Analysis/AnalyzeHotspots/calendar.png"
-                width: 22
-                height: width
-
-                MouseArea {
-                    anchors.fill: parent
-                    onClicked: {
-                        calendar.selectedDate = new Date(fromThisDate);
-                        calendarOverlay.toOrFromDate = "to";
-                        calendarOverlay.visible = true;
-                    }
-                }
+            text: "31 Jan 1998"
+            onTextChanged: {
+                toThisDate = Date.fromLocaleString(Qt.locale(), text, "d MMM yyyy")
```

Comment:
Should this be `dd MMM yyy`?

Reply:
Good question - I think that may expect dates to be formatted like "01 Jan 1970"
The original sample had them as "d MMM yyyy" and I kept it like that: 
https://github.com/Esri/arcgis-runtime-samples-qt/blob/v.next/ArcGISRuntimeSDKQt_QMLSamples/Analysis/AnalyzeHotspots/AnalyzeHotspots.qml#L166